src/subcommand/server.rs

```diff
@@ -1068,6 +1048,43 @@ impl Server {
       HeaderValue::from_static("public, max-age=31536000, immutable"),
     );
 
+    headers.insert(
+      header::CONTENT_TYPE,
+      inscription
+        .content_type()
+        .and_then(|content_type| content_type.parse().ok())
+        .unwrap_or(HeaderValue::from_static("application/octet-stream")),
+    );
+
+    if let Some(content_encoding) = inscription.content_encoding() {
+      if accept_encoding.is_acceptable(&content_encoding) {
+        headers.insert(header::CONTENT_ENCODING, content_encoding);
+      } else if server_config.decompress
+        && content_encoding
+          .to_str()
+          .map_err(|err| ServerError::Internal(err.into()))?
+          .to_lowercase()
+          == "br"
```

Comment:
I think we can just do:

```suggestion
        && content_encoding == "br"
```